samples/kafka/quickstart/consumer/src/main/resources/consumer.config

```diff
@@ -1,4 +1,4 @@
-bootstrap.servers={YOUR.EVENTHUBS.FQDN}:9093
+bootstrap.servers={EVENT HUB NAMESPACE}.servicebus.windows.net:9093
```

Comment:
These are the same thing

@ShubhaVijayasarathy is there a preferred way of referencing this between FQDN and namespace? If namespace, I'll change it throughout the repo